spec/variables_spec.js

```diff
@@ -47,4 +47,18 @@ describe('Variables', function () {
     ]);
     done();
   });
+
+  it('should resolve to string', function (done) {
+    ctx.templateSrv.replace = function(target, vars) {
```

Comment:
Hmm strictly speaking you're right but I prefer to keep here the more verbose form, to make it clear we expect "$container" and not anything else, to keep the test easy to understand.